addon/components/o-s-s/attributes-panel.stories.js

```diff
@@ -46,6 +46,16 @@ export default {
       },
       control: { type: 'boolean' }
     },
+    initialMode: {
+      description: 'Automatically enable edit mode',
+      table: {
+        type: {
+          summary: 'string'
```

Comment:
There's a way to have list the two options available so that they are better documented in storybook (+ they should work in the component preview in there too !)
There's an example [here](https://github.com/upfluence/oss-components/blob/master/addon/components/o-s-s/star-rating.stories.js#L46-L51)